doc/whats_new/v0.23.rst

```diff
@@ -60,6 +60,15 @@ Changelog
   :pr:`16149` by :user:`Jeremie du Boisberranger <jeremiedbb>` and 
   :user:`Alex Shacked <alexshacked>`.
 
+- |API| The ``n_jobs`` parameter of :class:`cluster.KMeans`,
+  :class:`cluster.SpectralCoclustering` and
+  :class:`cluster.SpectralBiclustering` is deprecated. :pr:`11950` by
+  :user:`Jeremie du Boisberranger <jeremiedbb>`.
+
+- |API| The ``precompute_distances`` parameter of :class:`cluster.KMeans` is
+  deprecated. It has no effect. :pr:`11950` by
+  :user:`Jeremie du Boisberranger <jeremiedbb>`.
+
```

Comment:
Yes I haven't made the what's new yet (only the deprecated stuff). Since this pr change includes fix, enhancement, efficiency, do you think I should add separate entries or I can fill a single entry with several tags ?

Reply:
Maybe a compound entry like we did for the HGBDT, e.g.

- Many improvements where made to KMEANS and co:
  - |FEA| ...
  - |FIX| ...
  - etc

Reply:
I only had 1 efficiency and one enhancement so I made 2 entries. I found that making a bullet list with 2 bullets was sad :)